geode-core/src/main/java/com/gemstone/gemfire/internal/cache/wan/serial/SerialGatewaySenderQueue.java

```diff
@@ -979,15 +1009,14 @@ private void initializeRegion(AbstractGatewaySender sender,
         SerialGatewaySenderQueueMetaRegion meta = new SerialGatewaySenderQueueMetaRegion(
             this.regionName, ra, null, gemCache, sender);
         try {
-          this.region = gemCache.createVMRegion(
-              this.regionName,
-              ra,
-              new InternalRegionArguments().setInternalMetaRegion(meta)
-                  .setDestroyLockFlag(true).setSnapshotInputStream(null)
-                  .setImageTarget(null)
-                  .setIsUsedForSerialGatewaySenderQueue(true)
-                  .setSerialGatewaySender(sender));
-          
+          this.region = gemCache.createVMRegion(this.regionName, ra, new InternalRegionArguments().setInternalMetaRegion(meta)
+                                                                                                  .setDestroyLockFlag(true)
+                                                                                                  .setSnapshotInputStream(null)
+                                                                                                  .setImageTarget(null)
+                                                                                                  .setIsUsedForSerialGatewaySenderQueue(true)
+                                                                                                  .setInternalRegion(true)
```

Comment:
I would be nice if this diff just showed one line (the addition of ".setInternalRegion(true)". Are you not using the checked in geode code style? I actually like your new indentation better but it seems like we should all use a consistent formatter.